styles.css

```diff
@@ -0,0 +1,197 @@
+nav {
+  position: relative;
+  z-index: 99 !important;
+  background-color: #fff;
+}
+
+.nav1 {
+  width: 97%;
+  margin: auto;
+  background-color: #f72210;
+  color: #fff;
+}
+
+.nav1 p {
+  margin: auto 30px;
+}
+
+.title {
+  color: #fff;
+  font-size: 90px;
+  font-weight: bold;
```

Comment:
- [ ] use `font-weight-bold` bootstrap class for this.

Reply:
- [ ] As was mentioned before, you should use bootstrap class for this.